src/react-components/src/components/Link/Link.tsx

```diff
@@ -1,11 +1,30 @@
-// Base Link Helper Class
 import * as React from "react";
 import { uid } from "react-uid";
+import { LinkTypes, LinkIconPositions } from "./LinkTypes";
 
 export interface LinkProps {
```

Comment:
since you're working on it, this is probably a good time to add `LinkOptions` to be exported and not export `LinkProps`

Reply:
Thanks for catching that. I'll review all your docs and maybe hit you up with some questions, I only glanced at everything yesterday.